src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,28 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+
+        String name1 = "firstBulldozer";
+        String color1 = "Yellow";
+        Bulldozer firstBulldozer = new Bulldozer(name1, color1);
```

Comment:
```suggestion
        Bulldozer firstBulldozer = new Bulldozer("firstBulldozer", "Yellow");
```
use them directly, no need to save strings into local variables if they are used immediately and not once since then. Also, naming with numbers is a sign something went wrong